### PR TITLE
Add support for mongo metrics in envoy bootstrap

### PR DESCRIPTION
**Pull request by douglas-reid**
This PR addresses #9660 .

I don't have a good way to test this at the moment. I'm hoping someone else can verify it works as intended before I spend time updating the tests.

The regex is built based on: https://www.envoyproxy.io/docs/envoy/latest/configuration/network_filters/mongo_proxy_filter#statistics

cc: @StefanCenusa

### CODE REVIEW

**Comment by douglas-reid** [comment]
Updated `delay_` to `delays_`, based on feedback from @StefanCenusa.

**Comment by douglas-reid** [comment]
Ping.

File: tools/deb/envoy_bootstrap_v2.json [hunk 1]
```diff
@@ -39,6 +39,10 @@
       {
         "tag_name": "listener_address",
         "regex": "^listener\\.(((?:[_.[:digit:]]*|[_\\[\\]aAbBcCdDeEfF[:digit:]]*))\\.)"
+      },
+      {
+        "tag_name": "mongo_prefix",
+        "regex": "^mongo\\.(.+?)\\.(collection|cmd|cx_|op_|delays_|decoding_)(.*?)$"
       }
```
Comment (rshriram) [inline tools/deb/envoy_bootstrap_v2.json]: Can we preemptively add mysql and Kafka please?

Comment (douglas-reid) [reply tools/deb/envoy_bootstrap_v2.json]: Do you have a way to test / validate those filters?  I'm happy to add whatever is needed, but I lack a good validation setup.

**Pull request merged**